.github/assets/check_no_std.sh

```diff
@@ -3,21 +3,21 @@ set -eo pipefail
 
 # TODO
 no_std_packages=(
-#   reth-codecs
+  reth-codecs
+  reth-primitives-traits
+#   reth-primitives
 #   reth-consensus
 #   reth-db
 #   reth-errors
 #   reth-ethereum-forks
 #   reth-evm
 #   reth-evm-ethereum
 #   reth-network-peers
-#   reth-primitives
-#   reth-primitives-traits
 #   reth-revm
 )
 
 for package in "${no_std_packages[@]}"; do
-  cmd="cargo +stable build -p $package --target riscv32imac-unknown-none-elf --no-default-features"
+  cmd="cargo +stable build -p $package --target wasm32-wasi --no-default-features"
```

Comment:
```suggestion
  cmd="cargo +stable build -p $package --target wasm32-wasip1 --no-default-features"
```
`wasm32-wasi` is being renamed to `wasm32-wasip1` and will be removed
https://github.com/rust-lang/compiler-team/issues/607